### ISSUE TITLE
1.0.16 really slow

### ISSUE THREAD

**Issue by ghost** [RocketChat/Rocket.Chat.Android#387]
Hello Devs.

I compile rocketchat myself. The lastet builds (maybe from 13.8. on?) are really slow. Are you aware of that? Switched back to a build from 24.7. which runs blazing fast.

thanks and cheers

**Comment by philipbrito** [comment]
We are not facing any performance issue w/ 1.0.16. Could you provide more info about that?

**Comment by ghost** [comment]
I am on development branch and compile rocketchat with ./gradlew assembleDebug.

```
himbeere@dev ~/Rocket.Chat.Android $ git status 
On branch develop
Your branch is up-to-date with 'origin/develop'.
nothing to commit, working tree clean
himbeere@dev ~/Rocket.Chat.Android $
```

After installing rocketchat needs 5 sec to authenticate and ~10sec to load a channel. Switching channels needs 10sec too. With a build from 24.7. everything is much faster. Authenticating takes 1sec and loading a channel needs 2sec.

**Comment by philipbrito** [comment]
Hello @himbeere 
We released [1.0.16](https://github.com/RocketChat/Rocket.Chat.Android/releases/tag/v1.0.16) minutes ago. Could you test it to give us a feedback? I think that your issue is now gone.

**Comment by ghost** [comment]
Indeed, it's even faster then before. Thanks.

cheers
t.
